wire-ios/Wire-iOS/Sources/UserInterface/Conversation/Content/Cells/ConfigurationMessageCell/Components/EphemeralCountdownView.swift

```diff
@@ -25,6 +25,7 @@ class EphemeralCountdownView: UIView {
     fileprivate let destructionCountdownView: DestructionCountdownView = DestructionCountdownView()
     fileprivate let containerView =  UIView()
     fileprivate var timer: Timer?
+    fileprivate var everAnimated = false
```

Comment:
I don't get the naming, do you mean alwaysAnimated ?

Reply:
It holds true if the animation was already fired once for this view.